src/tools/rbd_wnbd/rbd_wnbd.h

```diff
@@ -69,6 +70,9 @@ struct Config {
   bool remap_failure_fatal = false;
   bool adapter_monitoring_enabled = false;
 
+  bool wait_disk_online = false;
```

Comment:
Would anyone want a WNBD mapping that isn't online?  It seems like `--wait-online` behavior should be the default (perhaps even with a nice bonus of being able to output the Windows disk path on stdout like it's done on Linux)?